packages/shopify-codemod/test/fixtures/arguments-to-args-spread/nested-iife.input.js

```diff
@@ -0,0 +1,16 @@
+function foo() {
+  (function() { // Function IIFE has its own scope.
+    console.log(arguments, 'a' === args[0]);
+
+    (() => {
+      console.log(arguments, 'a' === args[0]);
+      (() => {
+        console.log(arguments, 'a' === args[0]);
+
+        (function() {
+          console.log(arguments, 'd' === args[0]);
```

Comment:
Nitpick, but might make more sense to use `arguments` since `args` is what it will be renamed to after the transform.